migrate/v0_24/staking_rewards.md

```diff
@@ -0,0 +1,80 @@
+# Migrate Staking Reward Calculation to Endpoint
+
+Kava 15 (v0.25.x) is changing the mechanism for staking rewards, which will no longer be inflationary but will be paid out of the community module. In order to continue displaying expected yields or APYs to users, wallets and explorers will need to update.
+
+To accommodate an easy transition to these new changes, a new endpoint has been backported to v0.24.
+
+The endpoint calculates staking rewards for the current kava version and is forward compatible with future changes.
+
+All consumers who display yearly staking reward percentages are encouraged to migrate from the standard calculation to using the endpoint, as the standard calculation will no longer be accurate in the near-future.
+
+Endpoint: `/kava/community/v1beta1/annualized_rewards`
+Example Response:
+```json
+{
+  "staking_rewards": "0.203023625910000000"
+}
+```
+
+## Before Kava 15
+The staking APR is calculated the same way as other cosmos-sdk chains. Various parameters are fetched and then combined in this calculation:
+```
+staking_apr ≈ mint_inflation *
+    (1 - distribution_params.community_tax) *
+    (total_supply_ukava/pool.bonded_tokens)
+```
+
+_Note this doesn’t include transaction fees paid to stakers._
+
+Endpoints used:
+
+* https://api.data.kava.io/cosmos/mint/v1beta1/params
+* https://api.data.kava.io/cosmos/distribution/v1beta1/params
+* https://api.data.kava.io/cosmos/bank/v1beta1/supply/by_denom?denom=ukava
+* https://api.data.kava.io/cosmos/staking/v1beta1/pool
+
+Informational Endpoints
+
+* https://api.data.kava.io/cosmos/mint/v1beta1/inflation
+* https://api.data.kava.io/cosmos/mint/v1beta1/annual_provisions
+
+## After Kava 15
+Kava 15 will implement new staking rewards as ratified in this proposal: https://www.mintscan.io/kava/proposals/141. They will come into effect at the “switchover time” on 2023-12-31.
```

Comment:
I think technically switchover is 2024-01-01 or after 2023-12-31 UTC

Reply:
Yes let's specify the time zone UTC